### PR TITLE
 🦋 Allow admin to set a QR-code-less order on their be-BOP #1621

### PR DESCRIPTION
**Pull request by ithiame**

### CODE REVIEW

File: src/routes/(app)/order/[id]/+page.svelte [hunk 1]
```diff
@@ -294,6 +296,13 @@
 									/></a
 								>
 							{/if}
+							{#if payment.method === 'card'}
```
Comment (coyotte508) [inline src/routes/(app)/order/[id]/+page.svelte:299]:
```suggestion
							{#if payment.method === 'card' && !data.hideCbQrCode}
```

no need to render it in the DOM at all

File: src/lib/server/runtime-config.ts [hunk 2]
```diff
@@ -181,6 +181,8 @@ const baseConfig = {
 	],
 	contactModes: ['email', 'nostr'],
 	posTouchTag: [] as Tag['_id'][],
+	hideCbQrCode: false,
+	overwriteCbSvgColor: false,
```
Comment (coyotte508) [inline src/lib/server/runtime-config.ts:185]: `cb` is a french abbrevation. Use `creditCard` instead

**Review by coyotte508 — APPROVED** [review]

**Comment by Tirodem** [comment]
We still need to remove this text when we are QR code less :
image
And I don't find it in translation key.

**Comment by Tirodem** [comment]
Also, order-creditCard-svg should be used for Stripe too.

**Comment by Tirodem** [comment]
Otherwise, it's good.

**Comment by Tirodem** [comment]
https://github.com/B2Bitcoin/beBOP/issues/1632 for stripe logo

**Pull request merged**